LoopKit/UserAlert.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  UserAlert.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 4/8/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+/// Protocol that describes any class that handles Alert posting.
+public protocol UserAlertHandler: class {
+    /// Schedule the given alert for posting.
+    func scheduleAlert(_ alert: UserAlert)
+    /// Unschedule any pending alerts with the given identifier.
+    func unscheduleAlert(managerIdentifier: String, typeIdentifier: UserAlert.TypeIdentifier)
```

Comment:
unscheduleAlert and cancelAlert -> removeDeliveredAlerts and removePendingAlerts ?

Reply:
I also never liked `scheduleAlert`, so if we're renaming `unscheduleAlert`, I prefer `issueAlert`.  I'll make all 3 changes.

Reply:
I think it's schedule vs issue in the ios api because it includes a trigger that actually does the issuing.